anvill/include/anvill/Lifters/Options.h

```diff
@@ -144,6 +145,10 @@ class LifterOptions {
   // forwarding.
   bool store_inferred_register_values : 1;
 
+  // Instructs the RecoverStackFrameInformation function pass to zero-initialize
```

Comment:
Eventually lets expand this to an enum that lets us not store, store undef, store zero, or fill with "symbolic" tracking values.